### PR TITLE
Bug - core IR error message

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11205)

## Description
image

image

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by maimorag — COMMENTED** [review]
Looks great!
please add a unit test

**Review by RotemAmit — APPROVED** [review]
Nice!

File: Packs/Core/ReleaseNotes/3_0_51.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Fixed an issue where an error returned instead of an entry when using the **blocklist_files_command** and **allowlist_files_command** commands when all of the hashes had already been added to the block or allow list.
```
Comment (dantavori) [inline Packs/Core/ReleaseNotes/3_0_51.md:6]:
```suggestion
- Fixed an issue in the ***core-blocklist-files*** and ***core-allowlist-files*** commands where an error was returned when all of the hashes had already been added to the blocklist or the allowlist.
```

**Review by dantavori — APPROVED** [review]

**Comment by MLainer1** [comment]
this PR needs a force merge due to TPB knows failures with open Jira tickets:
image

`demisto_client.demisto_api.rest.ApiException: (500)
Reason: Internal Server Error
HTTP response body: {"message":"Internal Server Error"}`

**Pull request merged**